wagtail/admin/views/workflows.py

```diff
@@ -382,6 +383,19 @@ def remove_workflow(request, page_pk, workflow_pk=None):
         return redirect("wagtailadmin_explore", page.id)
 
 
+class TaskTitleColumn(TitleColumn):
+    cell_template_name = "wagtailadmin/workflows/includes/task_title_cell.html"
+
+    def get_cell_context_data(self, instance, parent_context):
+        context = super().get_cell_context_data(instance, parent_context)
+        context["active"] = instance.active
+        return context
```

Comment:
It seems `instance` is available in the context (from the base `Column.get_cell_context_data()`), so we can do `instance.active` in the template and avoid the override

Reply:
Good spot! Updated and rebased now.